spec/tree_spec.rb

```diff
@@ -1,14 +1,137 @@
 require 'rspec'
 require 'tree'
 
+def kill_tree(tree)
+  # The 12 here is based off of the rand_seed the below
+  12.times { tree.age! }
+end
+
+def new_test_tree
```

Comment:
`subject` is a better option depending on what you're trying to convey.
you could also use `let` or let!` to accomplish this as well.